recipes/dimod/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dimod" %}
+{% set version = "0.10.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dimod-{{ version }}.tar.gz
+  sha256: b854a5e633d65b836fc8b90d4fb1ef411843629e3dc49f237e6846060f7b6d38
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - setuptools >=46.4.0
```

Comment:
Are you should this doesn't belong in `host`?

Reply:
(same for wheel)

Reply:
Oh no you’re right my bad, once sec.